src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig

```diff
@@ -159,7 +159,7 @@
                   <div class="content-block">
                     <div class="content-title">
                       <p>{{ 'lbl.Drafts'|trans|ucfirst }}
-                        <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpDrafts'|trans|ucfirst }}"></abbr>
+                        <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpDrafts'|trans|ucfirst }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320